packages/ckeditor5-widget/src/utils.js

```diff
@@ -304,6 +304,19 @@ export function findOptimalInsertionPosition( selection, model ) {
 	return selection.focus;
 }
 
+/**
+ * Checks if the selection is on object.
+ *
+ * @param {module:engine/model/selection~Selection|module:engine/model/documentselection~DocumentSelection} selection
+ * @param {module:engine/model/schema~Schema} schema
+ * @returns {Boolean}
+*/
+export function checkSelectionOnObject( selection, schema ) {
+	const selectedElement = selection.getSelectedElement();
+
+	return selectedElement && schema.isObject( selectedElement ) || false;
```

Comment:
```suggestion
	return !!selectedElement && schema.isObject( selectedElement );
```

is simpler and does not make you think about the order of operators.